### PR TITLE
Update Slab Preset

### PR DESCRIPTION
**Pull request by Chikorita-Lover**
Updates the slab preset to allow placing on bottom faces and combining the top and bottom faces

### CODE REVIEW

**Review by solvedDev — CHANGES_REQUESTED** [review]
LGTM. Make sure to bump the extension version though so people can actually update to a version with the changes you've made :)

**Review by solvedDev — APPROVED** [review]
Oops 🙃

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 1]
```diff
@@ -17,40 +17,44 @@
...
+			"IDENTIFIER_NAME",
+			{ "validate": ["required"] }
+		]
 	],
 
 	"createFiles": [
 		[
 			"block.json",
-			"blocks/{{PRESET_PATH}}{{IDENTIFIER}}.json",
+			"BP/blocks/{{PRESET_PATH}}{{IDENTIFIER}}.json",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:30]: Remove "BP/"

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 2]
```diff
@@ -17,40 +17,44 @@
...
 			{ "inject": ["IDENTIFIER", "PROJECT_PREFIX", "PRESET_PATH"], "openFile": true, "packPath": "behaviorPack" }
 		],
 		[
-			"texture.png",
-			"textures/blocks/{{PRESET_PATH}}{{IDENTIFIER}}.png",
-			{ "inject": ["IDENTIFIER", "PRESET_PATH"], "packPath": "resourcePack" }
+			"doubleBlock.json",
+			"BP/blocks/{{PRESET_PATH}}double_{{IDENTIFIER}}.json",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:35]: Remove "BP/"

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 3]
```diff
@@ -17,40 +17,44 @@
...
+			{ "inject": ["IDENTIFIER", "PROJECT_PREFIX", "PRESET_PATH"], "packPath": "behaviorPack" }
+		],
+		[
+			"lootTable.json",
+			"BP/loot_tables/blocks/double_{{IDENTIFIER}}.json",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:40]: Remove "BP/"

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 4]
```diff
@@ -17,40 +17,44 @@
...
+			{ "inject": ["IDENTIFIER", "PROJECT_PREFIX"], "packPath": "behaviorPack" }
 		],
 		[
 			"geometry.json",
-			"models/blocks/{{IDENTIFIER}}.json",
-			{ "inject": ["IDENTIFIER"], "packPath": "resourcePack" }
+			"RP/models/blocks/slab.geo.json",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:45]: Remove "RP/"

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 5]
```diff
@@ -17,40 +17,44 @@
...
 	"expandFiles": [
-		[
-			"terrainTexture.json",
-			"textures/terrain_texture.json",
-			{ "inject": ["IDENTIFIER", "PROJECT_PREFIX", "PRESET_PATH"], "packPath": "resourcePack" }
-		],
 		[
 			"clientBlock.json",
-			"blocks.json",
+			"RP/blocks.json",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:52]: Remove "RP/"

File: packages/minecraftBedrock/preset/block/slab/manifest.json [hunk 6]
```diff
@@ -17,40 +17,44 @@
...
 			{ "inject": ["IDENTIFIER", "PROJECT_PREFIX"], "packPath": "resourcePack" }
 		],
 		[
 			"en_US.lang",
-			"texts/en_US.lang",
+			"RP/texts/en_US.lang",
```
Comment (Joelant05) [inline packages/minecraftBedrock/preset/block/slab/manifest.json:57]: Remove "RP/"

**Review by Joelant05 — CHANGES_REQUESTED** [review]
I've made some comments where there are some changes to be made. Since we have updated bridge. to respect the pack paths defined in the project config, we cannot assume the packs will be called "BP", "RP" etc. This change is supported in presets with the "packPath" property as a replacement for starting the destination path with "BP", "RP" etc.

**Comment by Chikorita-Lover** [comment]
Okay, I've gone and changed those. They weren't working as intended when I was testing the preset, and instead saved the files in the project's base folder.

**Comment by Joelant05** [comment]
Were you testing them on the latest dev build?

**Comment by Chikorita-Lover** [comment]
No, I tested it on the latest stable release (2.1.5).

**Comment by Joelant05** [comment]
Ah ok, that's why then. We've only recently made these changes on the dev build. This looks good to merge now.

**Review by Joelant05 — APPROVED** [review]

**Pull request merged**